src/abi/anchor.rs

```diff
@@ -157,7 +157,9 @@ fn idl_instructions(
                 vec![IdlAccountItem::IdlAccount(IdlAccount {
                     name: "dataAccount".to_string(),
                     is_mut: true,
-                    is_signer: func.has_payer_annotation(),
+                    /// With @payer annotation, the account is created on-chain; however, if a @seed
+                    /// is provided, then th
```

Comment:
This comment is incomplete.

Reply:
I don't know what happened, must have been distracted half-way through typing that! Fixed